rest/route/commit_queue_test.go

```diff
@@ -154,8 +156,20 @@ func (s *CommitQueueSuite) TestClearAll() {
 }
 
 func (s *CommitQueueSuite) TestEnqueueItem() {
+	s.NoError(db.ClearCollections(patch.Collection))
 	route := makeCommitQueueEnqueueItem(s.sc).(*commitQueueEnqueueItemHandler)
-	id := mgobson.NewObjectId().Hex()
+	id := "aabbccddeeff112233445566"
+	p := &patch.Patch{
+		Id:      mgobson.ObjectIdHex(id),
+		Project: "test",
+		Author:  "somebody",
+		Patches: []patch.ModulePatch{
+			{
+				Githash: "revision",
+			},
+		},
+	}
+	s.NoError(p.Insert())
```

Comment:
if the mock looks for the patch in its `CachedPatches` the test won't need to insert a patch in the db.